packages/ui/src/components/molecules/CardTapList.tsx

```diff
@@ -3,15 +3,19 @@ import { styled } from 'styled-components';
 
 import { Flex, Indicator, LangDisplay, Typography } from '../atoms';
 
+type VariantType = 'onboarding' | 'other';
```

Comment:
It's better not to couple the parameter names with the flows
```suggestion
type VariantType = 'default' | 'muted';
```